x-pack/plugins/security_solution/public/resolver/test_utilities/simulator/index.tsx

```diff
@@ -253,8 +266,14 @@ export class Simulator {
    * The titles and descriptions (as text) from the node detail panel.
    */
   public nodeDetailDescriptionListEntries(): Array<[string, string]> {
-    const titles = this.nodeDetailEntryTitle();
-    const descriptions = this.nodeDetailEntryDescription();
+    /**
+     * The details of the selected node are shown in a description list. This returns the title elements of the description list.
+     */
+    const titles = this.domNodes('[data-test-subj="resolver:node-detail:entry-title"]');
```

Comment:
Node-detail makes me think the information is physically tied with the node in some way in the view rather than in the panel, maybe `node-panel-detail`? Not really sure if that's necessarily better

Reply:
actually, ignore me. `node-detail` is fine.